### ISSUE TITLE
Example code crashes

### ISSUE THREAD

**Issue by mfeif** [adafruit/Adafruit_CircuitPython_IRRemote#30]
There's a line in the example that crashes the script:
```python
code = decoder.decode_bits(pulses, debug=False)
```

decode_bits() no longer has that keyword. Changing the code to this makes it work:
```python
code = decoder.decode_bits(pulses)
```

**Comment by tannewt** [comment]
Hi @mfeif, thanks for the issue! Would you like to make a PR to fix the example? That would be awesome and I'm happy to help.

**Comment by mfeif** [comment]
Turns out it was just the README file that was wrong. I made a PR: #33. 

I also removed a bit in the example across the board that was code that was never used.

**Comment by tannewt** [comment]
Thank you!

**Comment by FoamyGuy** [comment]
Fixed with #33.
